### ISSUE TITLE
Publish extension to open-vsx.org for VS Codium

### ISSUE THREAD

**Issue by chrjorgensen** [halcyon-tech/vscode-ibmi#126]
This extension is not available in the marketplace [open-vsx.org](https://open-vsx.org/) used by [VS Codium](https://github.com/VSCodium/vscodium).

Instructions are in place on how to publish an extension to both marketplaces and can be found on the [Open VSX Wiki](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions).

It would be nice for this free extension to be available to the telemetry-free version of VS Code.

**Comment by worksofliam** [comment]
Cool idea. I'll get around to it sometime. In the mean time, you can still download the vsx from the Marketplace.

**Comment by worksofliam** [comment]
@chrjorgensen Seems like the latest version of Code for IBM i is there already.

What is the status of this?

**Comment by chrjorgensen** [comment]
Open VSX has an option to pull an extension into their marketplace, and I made a [PR](https://github.com/open-vsx/publish-extensions/pull/348) in their repository to pull this extension while waiting for you to decide whether to publish it yourself. Open VSX will also update the version periodically to the latest version.

I still prefer you to publish to both marketplaces - since the extension in Open VSX is always behind until refreshed. You can take ownership of the namespace and the extension as explained in the [wiki](https://github.com/eclipse/openvsx/wiki/Publishing-Extensions).

However, if you do not bother and just close this issue, we'll have to live with that.

TIA.

**Comment by worksofliam** [comment]
@chrjorgensen Thanks for letting me know. I will leave it open and I will be sure to come around to it.

**Comment by chrjorgensen** [comment]
@worksofliam Any thoughts on publishing to the Open VSX marketplace? This could be automated after claiming the namespace for your extension as @barrettotte had to do in.

It would be great to have your super extension published automatically to Open VSX - together with the additional extensions for IBM i which rely on this one. ❤️

**Comment by worksofliam** [comment]
@chrjorgensen - thanks for poking me again.

https://github.com/EclipseFdn/open-vsx.org/issues/813

Action has been updated, 0.8.20 will be published when it's done.

**Comment by worksofliam** [comment]
Looks like we're golden!

https://github.com/halcyon-tech/vscode-ibmi/runs/4862898839?check_suite_focus=true#step:7:1

**Comment by chrjorgensen** [comment]
Hi @worksofliam 

Did you write to me about the sudden problem with publishing to Open VSX? I would swear I saw something, but I can't remember where... well, whatever.

The publish process has failed for 8 days now - which coincides with a [change](https://github.com/eclipse/openvsx/pull/403) in the ovsx CLI tool . Could this be the cause?

I saw that your other extensions like vscode-rpgle didn't have problem publishing to Open VSX. Their ci.yaml script is a bit different and does not install the ovsx CLI tool with npm, but simply run the ovsx publish command with npx. npx downloads the ovsx CLI tool before running the publish function, so no problem here.

Hopefully my observation can help in fixing the problem publishing to ovsx...

😃
